pipeline/transformer/transformer_sierra/src/test/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraAlternativeTitlesTest.scala

```diff
@@ -42,8 +42,8 @@ class SierraAlternativeTitlesTest
   it("should extract all alternative titles when multiple fields defined") {
     val varFields = List(field130, field240, field246)
     getAlternativeTitles(varFields) shouldBe List(
-      "Apples",
       "Bananas",
+      "Apples",
```

Comment:
This is an example of the improved ordering logic – the titles in this test are

```
240 -> Apples
130 -> Bananas
246 -> Cherries
```

Now we retrieve them in MARC order, not transformer implementation order!